docs/SECURITY.md

```diff
@@ -7,3 +7,5 @@ layout: default
 # Reporting of security vulnerabilities
 
 If you discover a security vulnerability, we'd appreciate a non-public disclosure. The [issue tracker](https://github.com/systemd/systemd/issues) and [systemd-devel mailing list](https://lists.freedesktop.org/mailman/listinfo/systemd-devel) are fully public. If you need to reach systemd developers in a non-public way, report the issue to the [systemd-security@redhat.com](mailto:systemd-security@redhat.com) mailing list. The disclosure will be coordinated with distributions.
+
+To subscribe to [systemd-security@redhat.com](mailto:systemd-security@redhat.com) please submit subscribe request, but also reach out to the list administrators (@keszybz, @msekletar and @lnykryn) and send us a short justification why you need access to the list.
```

Comment:
*a* subscribe request

We would have to specify how to "reach out". I don't think we want people to open issues on github and cc us. So maybe we should have a template (something like what I pasted on telegram) and simply send it to a reply to any subscription request?